draft-ietf-tls-tls13.md

```diff
@@ -5568,6 +5709,48 @@ application protocols separately ensuring that confidential
 information is not inadvertently leaked.
 
 
+## Replay Attacks on 0-RTT {#replay-0rtt}
+
+Replayable 0-RTT data presents a number of security threats to
+TLS-using applications. Specifically, if applications are not
+engineered to be replay safe (minimally, this means idempotent, but
+in some cases may require even stronger conditions),
+then duplication of actions
+may cause side effects (e.g., purchasing an item or transferring
+money) to be duplicated, thus harming the site or the user.
+In addition, if data can be replayed a large number of times,
+this enables a variety of attacks via side channels such
+as cache timing or measuring the speed of cryptographic
+operations {{Mac17}}.
+
+The limited anti-replay described in {{anti-replay}} are intended to
```

Comment:
"anti-replay" is singular and would take "is", but I think this is supposed to be "anti-replay mechanisms".